src/config/dynamic-theme-fixes.config

```diff
@@ -2701,6 +2701,16 @@ header .logo
 
 ================================
 
+chem.libretexts.org
+
+CSS
+.internal {
+    background-image: none !important;
+    background-color: ${black} !important;
+}
```

Comment:
What about:

```

INVERT
.internal
```
?

Reply:
It is working better. It gives a dark background with a proper image. I will change that in the pr.